### PR TITLE
Features/results

### PR DESCRIPTION
**Pull request by tblattel**
Result input page that also shows last games

### CODE REVIEW

File: src/components/ResultsInput.vue [hunk 1]
```diff
@@ -1,69 +1,268 @@
...
+                team.games.length < tournament.gameLimit)
+            "
+          >
+            <form action="#" method="POST" @submit.prevent="checkForm">
+              <!-- <p class="capitalize-first" v-if="errors.length">
+                <b>{{ $tc("msgCorrectTheFollowingError", errors.length) }}:</b>
+                <ul>
+                  <li v-for="error in errors" v-bind:key="error">{{ error }}</li>
+                </ul>
+              </p> -->
```
Comment (ttirtia) [inline src/components/ResultsInput.vue]: Could you clean this up ?

File: src/components/ResultsInput.vue [hunk 2]
```diff
@@ -1,69 +1,268 @@
...
+          <div class="md:col-span-1">
+            <div class="px-4 sm:px-0">
+              <h3
+                class="text-lg font-medium leading-6 text-gray-900 text-left capitalize-first"
+              >
+                {{ $tc("previousGames", this.team.games.length) }}
+              </h3>
+              <!-- <p class="mt-1 text-sm text-gray-600 text-left capitalize-first">
+                {{ $t("msgResultsInput") }}
+              </p> -->
```
Comment (ttirtia) [inline src/components/ResultsInput.vue]: Same

File: src/components/Tournaments.vue [hunk 3]
```diff
@@ -1,60 +1,58 @@
...
+    </header>
+    <div
+      class="p-10 grid grid-cols-1 sm:grid-cols-1 md:grid-cols-3 lg:grid-cols-3 xl:grid-cols-3 gap-5"
+    >
+      <div
+        v-for="tournament in tournaments"
+        v-bind:key="tournament.id"
+        class="rounded overflow-hidden shadow-lg"
+      >
+        <!-- <img class="w-min" src="https://via.placeholder.com/150" alt="Mountain"> -->
```
Comment (ttirtia) [inline src/components/Tournaments.vue]: Yup

File: src/components/Tournaments.vue [hunk 4]
```diff
@@ -1,60 +1,58 @@
...
+        <a :href="'/table/' + tournament.id">
+          <div class="px-6 py-4">
+            <div class="font-bold text-xl mb-2">{{ tournament.name }}</div>
+            <p class="text-gray-700 text-base mt-8">
+              <!-- {{tournament.description}} -->
```
Comment (ttirtia) [inline src/components/Tournaments.vue]: Hum hum

File: src/components/Table.vue [hunk 5]
```diff
@@ -1,434 +1,287 @@
...
   components: {
     LayoutDefault,
   },
+  beforeMount() {
+    this.apiFindTournaments({
+      filter: { id: this.$route.params.id },
+    }).then((data) => {
+      this.tournament = data.findTournaments[0];
+      data.findTournaments[0].teams.flat().forEach((team) => {
+        team.players.flat().forEach((player) => {
```
Comment (ttirtia) [inline src/components/Table.vue]: Is `.flat()` really necessary ? Maybe I'm missing something

File: src/components/Table.vue [hunk 6]
```diff
@@ -1,434 +1,287 @@
...
   },
+  computed: {
+    sortedTeams: function(){
+      function compare(a, b) {
+        if ((a.placement ? a.placement : 1000) < (b.placement ? b.placement : 1000))
+          return -1;
+        if ((a.placement ? a.placement : 1000) > (b.placement ? b.placement : 1000))
+          return 1;
+
+        return 0;
```
Comment (ttirtia) [inline src/components/Table.vue]: You might get the same result with `return (a.placement || Number.MAX_SAFE_INTEGER) - (b.placement || Number.MAX_SAFE_INTEGER);`

File: src/layouts/LayoutDefault.vue [hunk 7]
```diff
@@ -76,19 +76,19 @@
                   href="/"
                   v-bind:class="[
                     currentRouteName == 'home'
-                      ? 'bg-gray-900 text-white block px-3 py-2 rounded-md text-base font-medium'
-                      : 'text-gray-300 hover:bg-gray-700 hover:text-white block px-3 py-2 rounded-md text-base font-medium',
+                      ? 'bg-gray-900 text-white block px-3 py-2 rounded-md text-base font-medium capitalize'
```
Comment (ttirtia) [inline src/layouts/LayoutDefault.vue]: `capitalize` might not be the best choice

**Review by ttirtia — CHANGES_REQUESTED** [review]
Looking good, just a few things that might be improved a bit.

File: src/components/ResultsInput.vue (hunk 1 above)
Comment (tblattel) [reply src/components/ResultsInput.vue]: done

File: src/components/ResultsInput.vue (hunk 2 above)
Comment (tblattel) [reply src/components/ResultsInput.vue]: done

File: src/components/Tournaments.vue (hunk 3 above)
Comment (tblattel) [reply src/components/Tournaments.vue]: done

File: src/components/Table.vue (hunk 6 above)
Comment (tblattel) [reply src/components/Table.vue]: Genius.

**Review by ttirtia — APPROVED** [review]
This is wonderful

**Pull request merged**